project/Build.scala

```diff
@@ -14,6 +14,10 @@ object Scrooge extends Build {
   val suffix = if (branch == "master") "" else "-SNAPSHOT"
 
   val libVersion = "3.18.1" + suffix
+
+  // For develop users: if you are using new features in util and finagle dependencies
```

Comment:
Anyway, I think that explanation is already included at https://github.com/twitter/scrooge/blob/master/README.md